packages/kobotoolbox/test/index.js

```diff
@@ -1,14 +1,16 @@
 import chai from 'chai';
 const { expect } = chai;
 import { enableMockClient } from '@openfn/language-common/util';
+import { request } from '../src/Utils';
 
-import nock from 'nock';
+import { http } from '../src';
```

Comment:
Can you remove `nock` now and then we can close #984?

From `packages/kobotoolbox` I think it;'s just `pnpm remove nock`, and then check in the diff